packages/models-library/requirements/_test.in

```diff
@@ -9,21 +9,21 @@
 --constraint _base.txt
 
 # testing
-coverage
+
 pytest
 pytest-aiohttp  # incompatible with pytest-asyncio. See https://github.com/pytest-dev/pytest-asyncio/issues/76
-pytest-cov
 pytest-icdiff
 pytest-instafail
 pytest-runner
 pytest-mock
 pytest-sugar
 
-# tools
-pylint  # NOTE: The version in pylint at _text.txt is used as a reference for ci/helpers/install_pylint.bash
+pytest-cov
```

Comment:
I am a bit confused about the relationship between `pytest-cov` and `coverage`, are both needed?

Reply:
``pytest-cov`` is a pytest extension that it also includes ``coverage``, this is the requiremenst.txt 

```text
...
coverage==5.5
    # via
    #   -r requirements.in
    #   pytest-cov
...
```